network/80-container-ve.network

```diff
@@ -24,3 +27,6 @@ LLDP=yes
 EmitLLDP=customer-bridge
 IPv6AcceptRA=no
 IPv6SendRA=yes
+
+[DHCPServer]
+PersistLeases=no
```

Comment:
As far as I can remember it was rejected a couple of times:
https://github.com/systemd/systemd/pull/31772#issuecomment-2000053357
https://github.com/systemd/systemd/pull/30021#discussion_r1518788578

because my nonsensical leases weren't convincing enough :-)

Reply:
Well, the /28 prefix is just too small to allow persistent leases here, because after a bunch of restarts the maximum is already exhausted...

Reply:
the other option would be to drastically lower the lease lifetime.

Reply:
Just to clarify I think it's the right default. I just wanted to bring those use cases again in hopes that those decisions can be finally reconsidered.

Reply:
I think it is ok to disable saving leases on a persistent storage. But even in such cases, I think it is better to serialize leases so that we can safely restart networkd. That's why I previously rejected @evverx's request. Now networkd has serialization/deserialization logic, hence we can easily implement that. I will work on that.

Reply:
> it is better to serialize leases

Dunno. If the config says that they shouldn't be persistent I'm not sure they should be serialized/deserialized (though that particular part is fuzzed so hopefully it shouldn't crash trying to deserialize bogus leases over and over again at least).

> we can safely restart networkd

networkd implements the logic where it tries to give out the same IP addresses so it's (kind of) persistent if certain conditions are met.